src/scss/layout/_navigation.scss

```diff
@@ -78,4 +81,61 @@
     align-items: flex-start;
     display: flex;
   }
+
+  @include viewport-desktop() {
+    &__main {
+      padding-left: 8%;
+    }
+
+    &__overlay-link {
+      display: inline-block;
+      height: 100%;
+      margin-right: 36px;
+      color: $gray;
+
+      &--active,
+      &:hover {
+        color: black;
+      }
+
+      li:last-of-type & {
+        margin-right: 0;
+      }
+    }
+
+    &__overlay-list {
+      display: flex;
+      font-size: $desktop-card-titles;
+      font-family: $hkgrotesk-bold;
+    }
+
+    & {
```

Comment:
Instead of creating its own selector, you can add it directly below
`@include viewport-desktop() {`
And it should work

Reply:
fix ok